### PR TITLE
Update for css formatting, ru-ru translation and small changes to templates

### PR DESCRIPTION
**Pull request by goonyaa**

### CODE REVIEW

File: public/css/style.css [hunk 1]
```diff
@@ -436,15 +473,15 @@ select#bottom_language_selector {
 
 .comment-date {
 	font-size: smaller;
-	width: auto; /* Undo bootstrap's fixed width */
+	/* Undo bootstrap's fixed width */
```
Comment (sfan5) [inline public/css/style.css]: Why move the comments? :confused:

Comment (goonyaa) [inline public/css/style.css]: Just don't like end of line comments. Can revert this part.

**Comment by akuma06** [comment]
Resolved conflicts.

File: public/css/style.css (hunk 1 above)
Comment (akuma06) [inline public/css/style.css]: I prefer end of line comments with // and new line comments with /* */. So I'm more for his way of doing it

**Pull request merged**